Gadget.Server/Authorization/AuthorizationController.cs

```diff
@@ -46,22 +47,31 @@
~        
~
~        [HttpPost("logout")]
~        public async Task<IActionResult> Logout(LoginRequest request)
~        {
             return Ok();
         }
 
-        [HttpPost]
```

Comment:
nazwa tej metody jest dziwna, gdy to czytam if ValidateToken == true to unauthorized? co ta metoda robi, bo z nazwy ciezko wyczuc